weave-js/src/config.ts

```diff
@@ -16,6 +17,7 @@ if (!window.WEAVE_CONFIG) {
     ANALYTICS_DISABLED: false,
     ONPREM: false,
     WEAVE_BACKEND_HOST: '/__weave',
+    WANDB_BASE_URL: '',
```

Comment:
perhaps we should default this to public wandb?

Reply:
done